generators/client/templates/react/src/main/webapp/app/modules/administration/administration.reducer.spec.ts.ejs

```diff
@@ -41,7 +41,6 @@ import administration, {
 } from './administration.reducer';
 
 describe('Administration reducer tests', () => {
-  const username = process.env.E2E_USERNAME ?? 'admin';
```

Comment:
Seems to be used for websocket tests.

Reply:
you're right, I fix that

Reply:
Move to line 229, can remove the condition

Reply:
yes

Reply:
Thanks.